plugins/modules/instance.py

```diff
@@ -851,6 +864,23 @@ def _update_instance(self) -> None:
         if should_update:
             self._instance.save()
 
+        ipv4_length = len(self.module.params.get("additional_ipv4") or [])
```

Comment:
```suggestion
        ipv4_length = len(self.module.params.get("additional_ipv4", []))
```
Optional suggestion. It might be more readable this way.

Reply:
I tried doing it this way and it didn't work. I think this is because `additional_ipv4` is an optional parameter, so if it isn't set `len` is run on a `NoneType`, causing the test to fail.

Reply:
Great, that makes sense!